assignment.sh

```diff
@@ -19,6 +19,55 @@ case $key in
 \tToday is $(date +"%A, %m %d %Y %H:%M") "
     echo
     ;;
+    
+    --mkdir|--make-directory)
+    MKDIR="$2"
+    shift #past argument
+    shift 
+    mkdir ${MKDIR}
+    echo
+    ;;
+    
+    -u|--to-uppercase)
+    U="$2"
+    shift #past argument
+    shift
+    echo $(echo $U | tr a-z A-Z ) 
+    echo
+    ;;
+    
+    -l|--to-lowercase)
+    L="$2"
+    shift #past argument
+    shift
+    echo $(echo $L | tr A-Z a-z ) 
+    echo
+    ;;
+    
+    -uf|--to-uppercase-first)
+    UF=$2
+    shift #past argument
+    shift
+    UF=`echo ${UF:0:1} | tr  '[a-z]' '[A-Z]'`${UF:1}
+    echo $UF
+    ;;
+    
+    -lf|--to-lowercase-first)
+    LF=$2
+    shift #past argument
+    shift
+    LF=`echo ${LF:0:1} | tr  '[A-Z]' '[a-z]'`${LF:1}
+    echo $LF
+    ;;
+    
+    --git-keep)
+    GITKEEP=$2
+    shift #past argument
+    
+    GITKEEP=`touch .gitkeep`
```

Comment:
You should check for passed argument. 
Now you are creating `.gitkeep` in current directory, you should be creating it in passed directory.
Example:
```bash
./arguments.sh --git-keep myFolder
```

and when i do `ls myFolder` i should see:
```bash
.
..
.gitkeep
```

Consider that we are ALWAYS passing argument